css/style.css

```diff
@@ -1,9 +1,213 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
-    font-family: 'Roboto', sans-serif;
+ 
+ body {
+    font-family: 'Roboto',
+    sans-serif;
     -webkit-font-smoothing: antialiased;
+    background-color: white;
+     }
+/*General styles*/
+
+/*Navigation styles*/
+
+#header {
+    background-color: white;
+    border-bottom: 1px solid grey;
+    top: 0;
+    width: 100%;
+}
+
+#logo {
+    width: 30px;
+    padding-left: 200px;
+    float: left;
+}
+
+#nav-bar {
+    text-align: center;
+}
+
+#nav-bar ul {
+    list-style: none;
+    padding: none;
+    margin: none;
+}
+
+
+#nav-bar li {
+    display: inline-flex;
+    
+}
+
+#nav-bar li a {
+    color: grey;
+    text-decoration: none;
+    padding: 20px;
+   
+}
+
+#nav-bar li a:hover {
+    color: black;
+}
+
+#active-link {
+    font-weight: bold;
+    color: black;
+}
+
+/*Hero styles*/
+
+.hero-image {
+    width: 1980px;
```

Comment:
Don't set such a precise width here - if you want an element to always be the width of the screen, set it to `width: 100%`